po/de.po

```diff
@@ -12843,6 +12842,30 @@ msgstr " - DEBUG"
 msgid "Version: %s,%s Built: %s%s"
 msgstr "Version: %s,%s Erstellt: %s%s"
 
+# NullBot tooltip
+msgid "Adaptive AI with multiple personalities"
+msgstr "Anpassungsfähige KI mit mehreren Persönlichkeiten"
+
+# Hover AI tooltip
+msgid "Sea map AI, based on NullBot"
+msgstr "KI für Seekarten, basierend auf NullBot"
+
+# Turtle AI tooltip
+msgid "Tower wars AI, based on NullBot"
+msgstr "KI für Befestigungen, basierend auf NullBot"
+
+# Nexus tooltip
+msgid "Initial AI, now supplanted by NullBot"
+msgstr "Ursprüngliche KI, von NullBot verdrängt"
+
+# SemperFi JS tooltip
+msgid "Prototypical JS AI focusing on rockets/missiles"
+msgstr "Prototyp für JS KI mit Schwerpunkt (Lenk-)Raketen"
+
+# SemperFi tooltip
+msgid "Enhanced Nexus AI"
+msgstr "Verbesserte Nexus KI"
```

Comment:
See comments for the [other commit](https://github.com/Forgon2100/warzone2100/commit/ce49fbeffb489354cfac80efaa7c056c8f183447).

Reply:
Okay.